### PR TITLE
Patch/readme

### PR DESCRIPTION
**Pull request by kappapiana**
Some cleanup and clearer -- hopefully -- menu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,59 +35,45 @@ This is the second option.
...
 
-Suppose that you have installed (copied) it in `~/scripts/anonymize.sh` and that your documenti is in `~/documents/doc.odt`
+It should be sufficient to have a running python3 environment, the imported libraries are all from the standard set.
 
-Do this:
 
-```shell
-ls -l ~/scripts/anonymize.sh # (make sure it's "executable")
 ```
+[script_directory]/anonymize.py filename.[odt|docx]
```
Comment (yamanq) [inline README.md:54]: There is an extra newline here.

File: anonymize.py [hunk 2]
```diff
@@ -232,10 +230,11 @@ def unzip_file(orig_file, tmp_dir):
...
 def cycle_ask(cur_files):
     '''opens and reads file to be changed, asks for input until user is fine
-    then writes the changed string, until you call it quits. This is sort of the
-    main function here'''
+    then writes the changed string, until you call it quits. This is sort of
+    the main function here'''
 
     # Flatten list of lists structure
+    # FIXME filelist variable declared but not used. Missing anything?
     filelist = [item for subfile in cur_files for item in subfile.textfiles]
```
Comment (yamanq) [inline anonymize.py:238]: You're right, this can be removed. I think I forgot to take it out when I was testing a different approach

**Review by yamanq — COMMENTED** [review]
There are some grammar issues in the README, so if you are ok with it, I will push a commit to this branch that fixes those issues.

File: README.md [hunk 3]
```diff
@@ -8,20 +8,22 @@
 
 [![REUSE status](https://api.reuse.software/badge/github.com/kappapiana/anonymize)](https://api.reuse.software/info/github.com/kappapiana/anonymize)
 
+Skip to [howto python]
```
[the remark this answers is not in the document]
Comment (kappapiana) [reply README.md]: Fixed. I use a special pandoc conversion that does not require declaring the anchor, so it slipped in and my preview did not catch the markdown error.

File: README.md (hunk 1 above)
Comment (yamanq) [reply README.md:54]: I meant the newline right after `[script_directory]/anonymize.py filename.[odt|docx]`, which looks strange when rendered:

image

**Pull request merged**